modules/seqsero2/main.nf

```diff
@@ -0,0 +1,45 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process SEQSERO2 {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::seqsero2=1.2.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/seqsero2:1.2.1--py_0"
+    } else {
+        container "quay.io/biocontainers/seqsero2:1.2.1--py_0"
+    }
+
+    input:
+    tuple val(meta), path(seqs)
+
+    output:
+    tuple val(meta), path("results/*_log.txt")   , emit: log
+    tuple val(meta), path("results/*_result.tsv"), emit: tsv
+    tuple val(meta), path("results/*_result.txt"), emit: txt
+    path "versions.yml"                          , emit: versions
+
+    script:
+    def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    SeqSero2_package.py \\
+        $options.args \\
+        -d results/ \\
```

Comment:
```suggestion
        -d ./ \\
```
It's preferred to don't use custom name output folder. Would it work like this?

Reply:
Let's give it a shot here! Tried locally and it didn't like ./ (Had to do with symbolic link of input, which I think is seqsero2 related)